doc/installation.md

```diff
@@ -88,3 +88,28 @@ If you are using Spacemacs, you can easily install org-roam by creating a simple
 ```
 
 Next, append `org-roam` to the `dotspacemacs-configuration-layers` list in your `.spacemacs` configuration file. Reload (`SPC f e R`) or restart Emacs to load `org-roam`. It's functions are available under the prefix `SPC a r` and `, r` when visiting an org-mode buffer. 
+
+## Doom Emacs
+
+If you are using [Doom Emacs](https://github.com/hlissner/doom-emacs), configure packages as explained in the [getting started](https://github.com/hlissner/doom-emacs/blob/develop/docs/getting_started.org#configuring-packages) guide. 
+
+Declare org-roam as a package in your `~/.doom.d/packages.el`:
+
+```elisp
+;; ~/.doom.d/packages.el
+
+(package! org-roam
+  :recipe (:host github :repo "jethrokuan/org-roam" :branch "develop"))
```

Comment:
Is the `:branch` necessary, if `develop` is the default branch?

Reply:
for now yes, I'll probably change it again when I push out 1.0.0.